src/graphql/schema/Trip/typeDefs.gql

```diff
@@ -1,17 +1,21 @@
 extend type Query {
-  trip(tripID: ID!): Trip
+  trip(id: ID!): Trip
   allTrips: [Trip]!
 }
 
 extend type Mutation {
-  updateTrip(input: updateTripInput!): updatedTrip!
+  createTrip(trip: createTripInput!, userID: ID): Trip!
```

Comment:
Why is the userID not required? There has to be a user who created the trip ==> could prevent us from adding falsy values or queries to the DB

Reply:
`updateTrip` doesn't need a userID (for now) because we don't store the userID for changes.

Reply:
But I think it would be bettter to:
- [ ] extract the trip id out of the `createTripInput` and have it as a primary, mandatory parameter